setup.cfg

```diff
@@ -163,6 +168,7 @@ markers =
     aiobotocore
     kafka
     grpc
+addopts=--random-order
```

Comment:
I added this while debugging the test failures on the old flask tests. I think we should leave it on, to try to prevent adding any tests that rely on order.